src/Tools/Console/Command/RunSqlCommand.php

```diff
@@ -77,7 +77,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $forceFetch = $input->getOption('force-fetch');
         assert(is_bool($forceFetch));
 
-        if (stripos($sql, 'select') === 0 || $forceFetch) {
+        $fetchKeywords = ['select', 'show'];
```

Comment:
`show` is platform specific, right?

Reply:
Yes, only mysql work. Should I add some method in Platform class to determine it ?

Reply:
We want to avoid platform-specific code in generic classes. Since `SHOW` is a MySQL extension, sniffing for it here is a no-go.

Reply:
> Should I add some method in Platform class to determine it ?

I think it feels kinda weird to have a new method in platform classes for the sole purpose of supporting this very specific use case of a feature that is very questionable and secondary (using the DBAL to run arbitrary SQL input by the user), especially since people can use `--force-fetch` and get the correct result.

Reply:
Yeah, I don't think that we should add another layer of guesswork either. I'd rather like to remove the current special handling of `SELECT`, tbh.